project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject democracyworks/pedestal-toolbox "0.6.3-SNAPSHOT"
+(defproject democracyworks/pedestal-toolbox "0.6.3"
```

Comment:
PR's should leave `-SNAPSHOT` in the version. We only drop those when we roll a release.